backend/benefit/applications/tests/test_applications_api.py

```diff
@@ -2219,6 +2219,40 @@ def test_handler_application_filter_archived(handler_api_client):
     assert response.data[0]["archived"]
 
 
+def test_application_pdf_print(api_client, application):
+    settings.NEXT_PUBLIC_MOCK_FLAG = False  # noqa
+    reseed(12345)
+
+    # Can access own applications
+    response = api_client.get(
+        f"/v1/print/{application.id}/",
+        {"as_html": "1"},
+    )
+    assert response.status_code == 200
+
+
+def test_application_pdf_print_denied(api_client, anonymous_client):
+    settings.NEXT_PUBLIC_MOCK_FLAG = False  # noqa
+    reseed(12345)
+
+    application = DecidedApplicationFactory()
+
+    # Cannot access anonymously
+    response = anonymous_client.get(
+        f"/v1/print/{application.id}/",
```

Comment:
Here for example the named route could be used.
`reverse("print_application", kwargs={"id":  application.id}).`